src/prefabs/structures/RadioInput/options/index.ts

```diff
@@ -0,0 +1,18 @@
+import { option, toggle, variable } from '@betty-blocks/component-sdk';
+import { advanced } from './advanced';
+import { validation } from './validation';
+import { styles } from './styles';
+
+export const options = {
+  actionProperty: option('ACTION_JS_PROPERTY', {
+    label: 'Property',
+    value: '',
+  }),
+
+  label: variable('Label', { value: [''] }),
+  value: variable('Value', { value: [''] }),
+  required: toggle('Required'),
```

Comment:
The 'required' option should be placed under the advanced options. You can use this as reference, all input components should follow this pattern: https://bettyblocks.atlassian.net/wiki/spaces/PT/pages/3166142496/Input+-+Options+Ordering